src/bindings/python/src/pyopenvino/graph/model.cpp

```diff
@@ -701,9 +706,127 @@ void regclass_graph_Model(py::module m) {
         return "<" + class_name + ": '" + self.get_friendly_name() + "'\ninputs[\n" + inputs_str + "\n]\noutputs[\n" +
                outputs_str + "\n]>";
     });
+    model.def("get_rt_info",
+              (PyRTMap & (ov::Model::*)()) & ov::Model::get_rt_info,
+              py::return_value_policy::reference_internal,
+              R"(
+                Returns PyRTMap which is a dictionary of user defined runtime info.
+
+                :return: A dictionary of user defined data.
+                :rtype: openvino.runtime.RTMap
+             )");
+    model.def(
+        "get_rt_info",
+        [](const ov::Model& self, const py::list& path) -> py::object {
+            std::vector<std::string> cpp_args;
+            for (size_t i = 0; i < path.size(); i++) {
+                cpp_args.emplace_back(path[i].cast<std::string>());
+            }
+            return Common::utils::from_ov_any(self.get_rt_info<ov::Any>(cpp_args));
+        },
+        py::arg("path") = py::list(),
+        R"(
+                Returns runtime attribute.
+
+                :param path: List of strings which defines a path to runtime info.
+                :type path: List[str]
+
+                :return: A py::object
+                :rtype: py:object
```

Comment:
```suggestion
                :return: A runtime attribute.
                :rtype: Any
```
Small nit, Sphinx and Python would not understand what is `py:object`. It should be Python type, not C++ one. (Note that this `Any` is different than `ov::Any`, it's built-in typing annotation)

Reply:
But do we have any for python?